code/__DEFINES/fonts.dm

```diff
@@ -18,7 +18,3 @@
 // Font metrics bitfield
 /// Include leading A width and trailing C width in GetWidth() or in DrawText()
 #define INCLUDE_AC (1<<0)
-
-DEFINE_BITFIELD(font_flags, list(
-	"INCLUDE_AC" = INCLUDE_AC,
-))
```

Comment:
do you know what this does? why are you touching it

Reply:
even if we do not currently use it it still does something, there is no reason to clear it away